microsoft-edge/progressive-web-apps-chromium/how-to/service-workers.md

```diff
@@ -41,63 +73,282 @@ The lifecycle of a Service Worker consists of multiple steps, with each step tri
 1.  The browser downloads the JavaScript file, installs the Service Worker, and triggers the `install` event. You can use the `install` event to pre-cache any important and long-lived files, such as CSS files, JavaScript files, logo images, offline pages, and so on from your website.
 
     ```javascript
-    self.addEventListener( "install", function( event ){
-        console.log( "WORKER: install event in progress." );
+    self.addEventListener("install", event => {
+        console.log("WORKER: install event in progress.");
     });
     ```
 
 1.  The Service Worker is activated, which triggers the `activate` event.  Use this event to clean up outdated caches.
 
     ```javascript
-    self.addEventListener( "activate", event => {
-        console.log('WORKER: activate event in progress.');
+    self.addEventListener("activate", event => {
+        console.log("WORKER: activate event in progress.");
     });
     ```
 
 1.  The Service Worker is ready to run when the page is refreshed or when the user goes to a new page on the site. If you want to run the Service Worker without waiting, use the `self.skipWaiting()` method during the `install` event, as follows:
 
     ```javascript
-    self.addEventListener( "install", event => {
+    self.addEventListener("install", event => {
         self.skipWaiting();
         // …
     });
     ```
 
-1.  The Service Worker is now running.
+1.  The Service Worker is now running and can listen to `fetch` events.
 
 
 <!-- ====================================================================== -->
-## Using fetch in Service Workers
+## Pre-cache resources
 
-The main event that you use in a Service Worker is the `fetch` event.  The `fetch` event runs every time the browser attempts to access content within the scope of the Service Worker.
+When a user accesses your app for the first time, the app's Service Worker is installed. Use the `install` event in your Service Worker to detect when this occurs, and cache all the static resources your app needs. Caching your app's static resources, such as the HTML, CSS, and JavaScript code needed by the start page, makes it possible to run your app even when the user's device is offline.
```

Comment:
```suggestion
When a user accesses your app for the first time, the app's Service Worker is installed. Use the `install` event in your Service Worker to detect when this occurs, and cache all the static resources your app needs. Caching your app's static resources (such as the HTML, CSS, and JavaScript code that are needed by the start page) makes it possible for your app to run even when the user's device is offline.
```
alt:
Caching your app's static resources (such as HTML, CSS, and JavaScript code) that are needed by the start page makes it possible for your app to run even when the user's device is offline.